san/transform.py

```diff
@@ -21,18 +25,33 @@ def transform_query_result(idx, query, data):
     If there is a transforming function for this query, then the result is
     passed for it for another transformation
     """
-    if query + '_transform' in globals():
+    if query in QUERY_PATH_MAP:
         result = path_to_data(idx, query, data)
-        globals()[query + '_transform'](result)
     else:
         result = data['query_' + str(idx)]
+
+    if query + '_transform' in globals():
+        globals()[query + '_transform'](result)
+
     return convert_to_datetime_idx_df(result)
 
 
 def eth_top_transactions_transform(data):
     for column in data:
         column['fromAddressIsExchange'] = column['fromAddress']['isExchange']
-        column['toAddressIsExchange'] = column['toAddress']['isExchange']
         column['fromAddress'] = column['fromAddress']['address']
+        column['toAddressIsExchange'] = column['toAddress']['isExchange']
         column['toAddress'] = column['toAddress']['address']
+        column['toAddress'] = column.pop('toAddress')
+        column['toAddressIsExchange'] = column.pop('toAddressIsExchange')
+        column['trxHash'] = column.pop('trxHash')
+        column['trxValue'] = column.pop('trxValue')
+    return data
+
+
+def news_transform(data):
```

Comment:
can we make the transform in a more functional programming way without `pop`.
something like this:
```python
def news_transform(data):
    return list(map(lambda column: {
       'title': column['title'],
       'description': column['description'],
       ...
    }, data))
```

Reply:
I think that will be easier to grasp :?

Reply:
You are right, I've made both of the functions this way now